api/pom.xml

```diff
@@ -286,6 +286,36 @@
                 </configuration>
             </plugin>
             
+            <!-- Generate the OSGi jar Manifest. -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
I'm getting
```
[WARNING] 'build.plugins.plugin.version' for org.apache.felix:maven-bundle-plugin is missing. @ line 290, column 21
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```

Reply:
If version 5.1.6 is to be used (the latest) note that it needs 2 extra instructions to prevent it from creating a corrupted jar:

```xml
<instructions>
     <_noimportjava>true</_noimportjava>
     <_runee>JavaSE-11</_runee>
</instructions>
```

Reply:
Sorry. It ran fine on my machine without that, so I was unaware of the need here. :)